### PR TITLE
[YGO-110] Adjust the ages filter in Activity Finder.

### PR DESCRIPTION
**Pull request by manachynskyi**
This PR is going to adjust filters on Activity Finder.
Currently, if the Ages filter on Activity Finder search results expanded, it collapses every time age is selected. 
Also removed jumping to the top of the page after choosing a filter, for preventing scrolling back to filters.

## Steps for review
- [ ] Create a new page and add **Activity Finder Search** paragraph.
- [ ] Visit page -> open Age (or some other filter) 
- [ ] make sure that filter is not collapsed after selecting items
- [ ] Make sure after choosing a filter, page doesn't scroll to top of page
- [ ] Close days filter, and check some value on Ages filter. While app will wait response Days tab shouldn't be expanded

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/custom/openy_activity_finder/templates/openy-activity-finder-program-search-page.html.twig [hunk 1]
```diff
@@ -122,7 +122,6 @@
                 expander_sections_config = '{{ expanderSectionsConfig|json_encode }}'
                 :default='$route.query.days'
                 v-on:updated-values="days = $event"
-                :loading="loading"
```
Comment (podarok) [inline modules/custom/openy_activity_finder/templates/openy-activity-finder-program-search-page.html.twig]: why loading removed?

**Review by podarok — CHANGES_REQUESTED** [review]

File: modules/custom/openy_activity_finder/templates/openy-activity-finder-program-search-page.html.twig (hunk 1 above)
Comment (manachynskyi) [reply modules/custom/openy_activity_finder/templates/openy-activity-finder-program-search-page.html.twig]: This option open **Days** filter each time when we wait for response from the controller. It lead to sidebar blinking.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Pull request closed without merging**